tests/test_request.py

```diff
@@ -443,9 +475,8 @@ def protocol(app):
 
     @app.route('/test', methods=['POST'])
     async def post(req, resp):
-        assert req.json == {}
-        resp.body = b'done'
+        assert req.json
```

Comment:
`raise` to make it clearer that we should not hit that piece of code?

Reply:
Actually, that's the instantiation that raises the error, maybe better to do `resp.body = req.json`?